controllers/api/api_status_controller.py

```diff
@@ -42,10 +42,13 @@ def _render(self):
         return json.dumps(status_dict, ensure_ascii=True)
 
     @classmethod
-    def clear_cache_if_needed(cls, old_content, new_content):
+    def clear_cache_if_needed(cls, old_content, new_content, users_not_to_notify=[]):
```

Comment:
Take a look here: https://github.com/the-blue-alliance/the-blue-alliance/pull/1313/files#diff-571c44c2d22fd07a2e30005f1e579956R123

This is where we send notifications for events down to users who have subscribed to that event. This will show a notification saying the event is down and clear the app-side cache of the API Status endpoint. A little farther down, we want the rest of the mobile clients to clear their cache, but I don't want to send duplicate notifications. Hence, the solution you see.

Can you think of a better way to do it? [Relevant app code](https://github.com/the-blue-alliance/the-blue-alliance-android/blob/master/android/src/main/java/com/thebluealliance/androidclient/gcm/GCMMessageHandler.java#L141)